src/BookNavigator/volumes/volumes-provider.js

```diff
@@ -0,0 +1,73 @@
+import { html } from 'lit-element';
+import sortAscendingIcon from '../assets/icon_sort_ascending.js';
+import sortDescendingIcon from '../assets/icon_sort_descending.js';
+
+import './volumes.js';
+
+export default class VolumesProvider {
+
+  constructor(optionChange, baseHost, bookreader) {
+    this.optionChange = optionChange;
+    this.component = document.createElement('viewable-files');
+
+    const files = bookreader.options.multipleBooksList?.by_subprefix;
+    this.viewableFiles = Object.keys(files).map(item => files[item]);
+    this.volumeCount = Object.keys(files).length;
+    this.isFirstLoad = true;
+    this.isSortAscending = false;
+
+    this.component.subPrefix = bookreader.options === undefined ? '' : bookreader.options.subPrefix;
+    this.component.hostUrl = baseHost;
+    this.component.viewableFiles = this.viewableFiles;
+
+    this.id = 'volumes';
+    this.label = `Viewable Files (${this.volumeCount})`;
+    this.icon = html`<ia-icon icon="volumes" style="width: var(--iconWidth); height: var(--iconHeight);"></ia-icon>`;
```

Comment:
suggestion: move out of `ia-icons` and directly use the volumes icon component.

(this will update your package.json & package-lock.json when it does - let's make sure the changes in those files will be scoped to this change)